### PR TITLE
Armor buffs, SSU buff, + a couple other changes.

### PR DESCRIPTION
**Pull request by ghost**
In this PR:

# Armor changes

Most armor is now significantly stronger. Nowhere near the specialized armor, but generally better. HOS and warden armor, in particular, are superior to regular security armor.

Hardsuits provide slightly more energy protection in general.

The standard wizard armor, while still available (as possible rewards or something idk), has been changed from the norm for the wizard to be able to buy, and provides extra, shielded protection, similar to syndicate shielded hardsuits (except with a longer recharge time.)

The same applies to the HOS hardsuit, except it is altogether weaker than either the syndicate or wizard shielded hardsuits.

Reactive armors also provide some armor bonuses where previously they provided nothing, and the ablative armor is better against energy protection in general.

# SSU Buff

Suit Storage units, when hacked to the super setting, can be used to cremate bodies similar to the crematorium. The way I see it, most SSU's are hard to get to anyway and the crematorium is altogether an easy way to get rid of bodies-it's probably harder to get someone into the suit storage unit and cremate them with it, especially considering they have ample time to scream (unless they're dead, at which point you can just cut off their head and remove their brain anyway)

If i'm wrong on this count and it gets abused i'll remove it.

# Other changes

Lich costs 4 spell points because 2op. That's about it tbh. If I missed anything lemme know.

I really really strongly suggest testmerging this, because i'm uncertain about how strong the armor buffs make armor. They shouldn't be absurdly overpowered but if it's too good, I'll close this.
##### Changelog

:cl:
rscadd: Suit Storage Units, when properly hacked, are capable of cremation instead of simple murder like they used to be.
rscadd: All armor has been (as an experiment) buffed. If this turns out to be too strong, it will be removed.
rscadd: Lich now costs 4 spell points over 2.
rscadd: The wizard's hardsuit is now a shielded variant. Have fun.
rscadd: The HOS's hardsuit now comes pre-equipped with a shield generator as well.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
don't give HoS overpowered shielded hardsuit unless it has like 1 charge and takes 20 years to recharge

**Comment by ghost** [comment]
it has 2 charges and takes a full minute to start recharging after being hit, and recharges half a shield every time afterwards

**Comment by Cruix** [comment]
If anyone needs a shielded hardsuit, it's the captain, and the captain does not need a shielded hardsuit.

**Comment by ghost** [comment]
cruix, purely experimental, and i don't see why having one shielded hardsuit on the station is a bad thing seeing as

a. armor is now better 

b. it can be beaten down easily with simply throwing things until the charge dissapears

the shielded hardsuit might be good on one on one, but 2v1 or more, it fails, and furthermore the hardsuit slows down

so unlike the syndicate shielded hardsuit which has no slowdown, the hos' slows him down indoors, has no jetpack in space and altogether easier to get through

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -609,8 +606,48 @@
...
+	icon_state = "hardsuit-hos"
+	name = "head of security's hardsuit"
+	desc = "A special bulky suit that protects against hazardous, low pressure environments. Has an additional layer of armor, and an inbuilt shield generator."
+	armor = list(melee = 45, bullet = 45, laser = 50, energy = 30, bomb = 25, bio = 100, rad = 50)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/security/hos
+	max_charges = 2
+	recharge_delay = 600
+	recharge_rate = 0.5 //How quickly the shield recharges once it starts charging
 
+////
```
Comment (ShadowDeath6) [inline code/modules/clothing/spacesuits/hardsuit.dm:650]: what

**Comment by ShadowDeath6** [comment]
p sure that sec hardsuits have inbuilt jetpacks now

also le flashbang spam is still a gg for being 2v1'ed

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -133,7 +133,7 @@
 	desc = "An ancient helmet made of bronze and leather."
 	flags_inv = HIDEEARS|HIDEHAIR
 	flags_cover = HEADCOVERSEYES
-	armor = list(melee = 25, bullet = 0, laser = 25, energy = 10, bomb = 10, bio = 0, rad = 0)
+	armor = list(melee = 35, bullet = 0, laser = -40, energy = 10, bomb = 10, bio = 0, rad = 0) //little known fact but the great fire of rome was actually caused by invading aliens
```
Comment (Notamaniac) [inline code/modules/clothing/head/helmet.dm:136]: -40 laser resist? I'm not sure where you can get this (probably some lavaland mob) but it really needs an addendum to the description that says it makes you very vulnerable to lasers.

**Comment by Super3222** [comment]
this calls for a testmerge

**Comment by Cruix** [comment]
I totally forgot about this PR

The wizard stuff and armor changes are fine but SSUs becoming crematoriums is a terrible idea and the HoS is not getting a shielded hardsuit, that is ridiculous.

**Pull request closed without merging**